src/hub/app.py

```diff
@@ -0,0 +1,127 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import os
+import sys
+
+import connexion
+import stripe
+import stripe.error
+from flask import current_app, g, jsonify
+from flask_cors import CORS
+from flask import request
+
+from sub.shared import secrets
+from sub.shared.cfg import CFG
+from sub.shared.exceptions import SubHubError
+from sub.shared.db import HubEvent
+from sub.shared.log import get_logger
+
+logger = get_logger()
+
+# Setup Stripe Error handlers
+def intermittent_stripe_error(e):
+    logger.error("intermittent stripe error", error=e)
+    return jsonify({"message": f"{e.user_message}"}), 503
+
+
+def server_stripe_error(e):
+    logger.error("server stripe error", error=e)
+    return (
+        jsonify({"message": f"{e.user_message}", "params": None, "code": f"{e.code}"}),
+        500,
+    )
+
+
+def server_stripe_error_with_params(e):
+    logger.error("server stripe error with params", error=e)
+    return (
+        jsonify(
+            {
+                "message": f"{e.user_message}",
+                "params": f"{e.param}",
+                "code": f"{e.code}",
+            }
+        ),
+        500,
+    )
+
+
+def server_stripe_card_error(e):
+    logger.error("server stripe card error", error=e)
+    return jsonify({"message": f"{e.user_message}", "code": f"{e.code}"}), 402
+
+
+def create_app(config=None):
+    logger.info("creating flask app", config=config)
+    region = "localhost"
+    host = f"http://localhost:{CFG.DYNALITE_PORT}"
+    stripe.api_key = CFG.STRIPE_API_KEY
+    if CFG.AWS_EXECUTION_ENV:
+        region = "us-west-2"
+        host = None
+    options = dict(swagger_ui=CFG.SWAGGER_UI)
+
+    app = connexion.FlaskApp(__name__, specification_dir=".", options=options)
+    app.add_api("swagger.yaml", pass_context_arg_name="request", strict_validation=True)
+
+    app.app.hub_table = HubEvent(table_name=CFG.EVENT_TABLE, region=region, host=host)
+
+    if not app.app.hub_table.model.exists():
+        app.app.hub_table.model.create_table(
+            read_capacity_units=1, write_capacity_units=1, wait=True
+        )
+
+    # Setup error handlers
```

Comment:
Is this comment necessary?  It looks straightforward.

Reply:
no, nuke it